internal/crypto/ptls/fips_strict.go

```diff
@@ -15,16 +14,16 @@ import (
 	"path/filepath"
 	"runtime"
 
-	"k8s.io/apiserver/pkg/server/options"
-
 	// Cause fipsonly tls mode with this side effect import.
 	_ "go.pinniped.dev/internal/crypto/fips"
 	"go.pinniped.dev/internal/plog"
 )
 
-// Always use TLS 1.2 for FIPs
-const secureServingOptionsMinTLSVersion = "VersionTLS12"
-const SecureTLSConfigMinTLSVersion = tls.VersionTLS12
+// goboring now also supports TLS 1.3 starting in Golang 1.21.6
```

Comment:
Glad this comment is here so if we need to understand later we can follow the trail. 👍